site/docs/operations/cluster-sizing.md

```diff
@@ -0,0 +1,114 @@
+
+Benchmark your jobs in a clustered setup to see the differences in performance, see the Sizing Example.
+
+## Sizing for failures
+
+Jet cluster is elastic to deal with failures and performance spikes. Mostly down-scales temporarily reduce the available resources and increase the stress on remaining members. The overal memory and CPU available in the cluster reduces. The data previously owned by the newly offline member is distributed among the remaining members. The cluster must catch up the missed data in the stream and keep up with the head of the stream.
+
+To tolerate the failure of one member, we recommend to size your cluster to operate with ```n-1``` members setup.
+
+You can use Hazelcast IMap Journal to ingest the streaming data. Journal is a in-memory structure with a fixed capacity. If the jobs consuming the journal can't keep up there is a risk of data loss.  The pace of the data producers and the capacity of the Journal therefore determine the lenght of an error window of your application. If you can't afford losing data, consider increasing the journal size or use persistent streaming storage such as Apache Kafka or Apache Pulsar.
+
+## Balancing cluster size with job count
+
+The jobs running in one cluster share the resources to maximise the HW utilization. This is efficient for setups without a risk of noisy neighbours such as:
+* Clusters hosting many short-living jobs
+* Clusters hosting jobs with a predictable performance 
+* Jobs with relaxed SLAs
+
+For stronger resource isolation (multi-tenant environments, strict SLAs), consider starting multiple smaller clusters with resources (CPU, memory, network) allocated on an OS level or a resource manager level.
+
+## Hardware Planning
+
+Jet is designed to run efficiently on homogeneous clusters. All JVM processes that participate in the cluster should have equal CPU, memory and network resources. One slow cluster member can kill the performance of the whole cluster.
+
+### Minimal Configuration
+
+Jet is a lightweight framework and is reported to run on a devices such as Rapsberry Pi Zerro (1GHz single-core CPU, 512MB RAM).
+
+### Recommended Configuration
+
+As a starting point for a data-intensive oprerations consider machines with: 
+
+* 8 CPU cores
+* 16 GB RAM
+* 10 Gbps network
+
+### CPU
+
+Jet can utilise hundreds of CPU cores efficiently by exploiting data and task parallelism. Adding more CPU can therefore help with scaling the CPU-bound computations. Read about the [Execution model](architecture/execution-engine.md) to understand how Jet makes the computation parallel and design your pipelines according to it. 
+
+Don't rely just on a CPU usage when benchmarking your cluster, benchmnark the throughput and latency instead. The task manager of Jet [can be configured](https://hazelcast.com/blog/idle-green-threads-in-jet/) to use the CPU aggresivelly. The idle Jet instance can use 20% of the CPU, Jet using 100% of the CPU can handle 5x more load.
```

Comment:
typo, aggressively

Reply:
"The idle Jet instance can use 20% of the CPU, Jet using 100% of the CPU can handle 5x more load"

This sounded confusing to me.